pkg/controller/secret/secret_controller.go

```diff
@@ -606,11 +606,14 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 	if fedrampVar, ok := os.LookupEnv("FEDRAMP"); ok {
 		fedramp, err := strconv.ParseBool(fedrampVar)
 		if err != nil {
-			reqLogger.Info("Unable to parse FEDRAMP environment variable, defaulting to %t.", fedramp)
+			reqLogger.Info("Unable to parse FEDRAMP environment variable, defaulting to %v.", fedramp)
```

Comment:
Does the `%v` template correctly?

Reply:
did not. it is fixed now. Tested with result like `{"level":"info","ts":1643664174.7293966,"logger":"secret_controller","msg":"FedRAMP environment","Request.Name":"prometheus-k8s-dockercfg-kwgfg","fedramp":true} `